src/extension.ts

```diff
@@ -249,6 +259,56 @@ export async function attach(
     } while (useRetry && Date.now() - startTime < timeout);
 }
 
+export async function attachToCurrentDebugTarget(context: vscode.ExtensionContext): Promise<void> {
+    if (!telemetryReporter) {
+        telemetryReporter = createTelemetryReporter(context);
+    }
+
+    telemetryReporter.sendTelemetryEvent('command/attachToCurrentDebugTarget');
+
+    // Attempt to attach to active CDP target
+    const session = vscode.debug.activeDebugSession;
+    if (!session) {
```

Comment:
can we add a test for this scenario? one where the mock value of:
vscode.debug.activeDebugSession = true;

Reply:
For sure!

Reply:
Done!